web/src/engine/websites/Baozimh_e2e.ts

```diff
@@ -16,9 +16,7 @@ const config = {
     },
     entry: {
         index: 5,
-        size: 164_832,
+        size: 151_930,
```

Comment:
The image is now smaller, therefore it seems replacing the CDN leads to lower quality images.

Reply:
I dont think the resolution is lower. The thing is : normal cdn has big watermarks 🤷

Reply:
Pictures resolutions are the same on both sides. Dont forget they are different (watermark/no watermark), so i assume the size differences are because of compression 🤷

Reply:
I was referring to smaller in size not in resolutions as there was a decrease by 13KB.
The missing watermark seems to explain why the new image is smaller in size.